### PR TITLE
Amazon price alert

### LINKED ISSUE #41: Amazon-price-alert
### Describe the new script you want
The script that alerts the customer when the price of an amazon product is below the threshold price value

### Describe the solution you'd like us to follow
Using web scraping we can make an alert for the customer to get the product when they get the fair deal 

## Do you want to work on this?
- [x] yes
- [ ] no

### PR DESCRIPTION
**Pull request by adityamittl**
# Description
The script that alerts the user if the amazon price of a product is below threshold price given by the user

Fixes #41 

## Type of change

Please delete options that are not relevant.

- [x] New Script (non-breaking change which adds a new script to the repository)

## Has This Been Tested?

- [x] I confirm that I have tested my script locally and verified that it works as intended.

## Checklist:

- [x] I have gone through the contributing guidelines before making this Pull Request.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: scripts/amazon-price-alert/script.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            """Your product <b>"""
+            + str(self.product_title)
+            + """</b> Found lower then your threshold value <b>"""
+            + str(self.budget)
+            + """</b>.\n This procuct is available is at  price of <b>"""
+            + str(self.product_price)
+            + """</b>"""
+        )
+        print(body)
+        dev_email = "Develope's Email Address"
```
Comment (OBITORASU) [inline scripts/amazon-price-alert/script.py:58]: Should be Developer's instead of Develop's

File: scripts/amazon-price-alert/script.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        dev_password = "Developer's Password"
+        server = smtplib.SMTP("smtp.gmail.com", "587")
+        server.ehlo()
+        server.starttls()
+        server.login(dev_email, dev_password)
+        server.sendmail(dev_email, self.email, body)
+        server.close()
+        print("Email Successfully send\n")
```
Comment (OBITORASU) [inline scripts/amazon-price-alert/script.py:66]: Should be sent instead of send.

**Review by OBITORASU — CHANGES_REQUESTED** [review]
A few typos check comments.

File: scripts/amazon-price-alert/script.py (hunk 1 above)
Comment (adityamittl) [reply scripts/amazon-price-alert/script.py:58]: The user email address is different which we asked them to enter, these are the credentials of the account of the person from whose account this email should send.

**Comment by adityamittl** [comment]
I fixed these typos

File: scripts/amazon-price-alert/script.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            + str(self.product_title)
+            + """</b> Found lower then your threshold value <b>"""
+            + str(self.budget)
+            + """</b>.\n This procuct is available is at  price of <b>"""
+            + str(self.product_price)
+            + """</b>"""
+        )
+        print(body)
+        dev_email = "Developer's Email Address"
+        dev_password = "Developer's Password"
```
Comment (OBITORASU) [inline scripts/amazon-price-alert/script.py:59]: Use User's instead of Developer's please, that sounds better.

**Review by OBITORASU — CHANGES_REQUESTED** [review]
Left a review

File: scripts/amazon-price-alert/script.py (hunk 3 above)
Comment (adityamittl) [reply scripts/amazon-price-alert/script.py:59]: Okay , Sure

**Review by OBITORASU — APPROVED** [review]
LGTM

**Pull request merged**